jwst/tests_nightly/general/nircam/#todo#

```diff
@@ -0,0 +1,13 @@
+strun assign_wcs.cfg wfs_in_1.fits
```

Comment:
Um... Should we be checking in `#todo#`? This does not seem legit. Is this a legit test file, or just some temporary file that no one cleaned up?

Reply:
I'll delete it.